steps/cli/oc.go

```diff
@@ -165,6 +166,16 @@ var _ = gauge.Step("Update addon config with resolverStepActions as <resolverSte
 	}
 })
 
+var _ = gauge.Step("Verify test versioned ecosystem task", func() {
+	taskList := cmd.MustSucceed("oc", "get", "task", "-n", "openshift-pipelines").Stdout()
+	requriedTasks := []string{"git-cli", "git-clone", "s2i-go", "s2i-python"}
+	for _, task := range requriedTasks {
+		if !strings.Contains(taskList, task) {
+			testsuit.T.Errorf("Failed to test versioned task, No task found!!")
```

Comment:
Provide context in error message ... which task was not found?

Reply:
Done